modules/video/test/test_trackers.cpp

```diff
@@ -126,4 +126,34 @@ TEST(DaSiamRPN, memory_usage)
     }
 }
 
+TEST(NanoTrack, memory_usage)
+{
+    cv::Rect roi(145, 70, 85, 85);
+
+    std::string backbonePath = cvtest::findDataFile("dnn/onnx/models/nanotrack_backbone_sim.onnx", false);
+    std::string neckheadPath = cvtest::findDataFile("dnn/onnx/models/nanotrack_head_sim.onnx", false);
+    cv::TrackerNano::Params params;
+    params.backbone = backbonePath;
+    params.neckhead = neckheadPath;
+    cv::Ptr<Tracker> tracker = TrackerNano::create(params);
+
+    string inputVideo = cvtest::findDataFile("tracking/david/data/david.webm");
+    cv::VideoCapture video(inputVideo);
+    ASSERT_TRUE(video.isOpened()) << inputVideo;
+
+    cv::Mat frame;
+    video >> frame;
+    ASSERT_FALSE(frame.empty()) << inputVideo;
+    tracker->init(frame, roi);
+    string ground_truth_bb;
+    for (int nframes = 0; nframes < 15; ++nframes)
+    {
+        std::cout << "Frame: " << nframes << std::endl;
+        video >> frame;
+        bool res = tracker->update(frame, roi);
+        ASSERT_TRUE(res);
```

Comment:
the test check nothing actually, because `update()` always return true in current implementation. It makes sense to record bboxes for first 15 frames and check the result against them as regression test.

Reply:
This part code is copied from the previous test case. Not sure if we should change this and the previous case.
https://github.com/opencv/opencv/blob/4.x/modules/video/test/test_trackers.cpp#L97-L127

Reply:
Please add bbox check or some other evidence that tracker does what expected. Current test just check if it does not crash, bot nothing for results.

Reply:
I got your point. How about adding two or three images test cases, the first one for the template and other images for object searching?

Reply:
Looks like the test bellow is enough and `memory_usage` may be safely removed. It does not add value.